adbfs

```diff
@@ -605,7 +608,7 @@ class Adb(object):
         if not dst.startswith('/'):
             dst = '/' + dst
 
-        cmd = [self.conf.adb_command, 'shell', 'mkdir', dst]
+        cmd = self._shell_cmd('mkdir', dst)
```

Comment:
That's peculiar. For listing files, you've used root (if available), but for files manipulation - no. So it might easily become an issue, that someone wanted to copy to/copy from/remove/mkdir some file, from directory which have different uid/gid than default user, so that it will result with errors. Is it done on some purpose? If so, please explain.

Reply:
I just converted whatever was there before. It probably makes sense to run mkdir as root, if possible.